skypy/neutral_hydrogen/__init__.py

```diff
@@ -1,4 +1,16 @@
-'''This module contains methods that model the properties of neutral hydrogen.
-'''
+"""
+This module contains methods that model the properties of neutral hydrogen
+emission in the Universe.
 
+Merger Rates
+================
 
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+
+   pr_halo_model
+
+"""
+
+from .mass import *  # noqa F401,F403
```

Comment:
It's ambiguous how the module is to be structured. In this way, you could access the function under both `skypy.neutral_hydrogen` as well as `skypy.neutral_hydrogen.mass`. If it's supposed to be the former, then I would add an underscore to `_mass` to indicate it is an implementation file. If the latter, then you should only `from . import mass`. (I guess it depends on how much stuff there is going to be in the module to decide which is right.)